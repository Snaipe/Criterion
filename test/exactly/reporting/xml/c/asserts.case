[act]

% asserts.c.bin --xml

[assert]

stdout
  -transformed-by ( replace_version | replace_times )
  equals -contents-of expected/asserts.txt
