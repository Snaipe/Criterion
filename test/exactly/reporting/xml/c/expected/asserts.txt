<?xml version="1.0" encoding="UTF-8"?>
<!-- Tests compiled with Criterion <VERSION> -->
<testsuites name="Criterion Tests" tests="8" failures="2" errors="0" disabled="0">
  <testsuite name="asserts" tests="8" failures="2" errors="0" disabled="0" skipped="0" time="<TIME>">
    <testcase name="wstring" assertions="0" status="PASSED" time="<TIME>">
    </testcase>
    <testcase name="string" assertions="0" status="PASSED" time="<TIME>">
    </testcase>
    <testcase name="stream" assertions="0" status="PASSED" time="<TIME>">
    </testcase>
    <testcase name="old_school" assertions="2" status="FAILED" time="<TIME>">
      <failure type="assert" message="2 assertion(s) failed."><![CDATA[asserts.c:19: ]]>&#10;<![CDATA[asserts.c:18: You can fail an assertion with a message from anywhere]]>&#10;</failure>
    </testcase>
    <testcase name="native" assertions="0" status="PASSED" time="<TIME>">
    </testcase>
    <testcase name="float" assertions="0" status="PASSED" time="<TIME>">
    </testcase>
    <testcase name="base" assertions="2" status="FAILED" time="<TIME>">
      <failure type="assert" message="2 assertion(s) failed."><![CDATA[asserts.c:13: This assert runs]]>&#10;<![CDATA[asserts.c:12: assert is fatal, expect isn't]]>&#10;</failure>
    </testcase>
    <testcase name="array" assertions="0" status="PASSED" time="<TIME>">
    </testcase>
  </testsuite>
</testsuites>
